pyiron_atomistics/table/funct.py

```diff
@@ -238,7 +238,8 @@ def get_equilibrium_parameters(job):
 
 
 def get_structure(job):
-    atoms = pyiron_to_ase(job.to_object().get_structure())
+    atoms = Atoms().from_hdf(job["output"])
```

Comment:
This is not the same, as the structure is not always allocated at the `job["output"]` For example for the `Murnaghan` job this does not work: 
https://github.com/pyiron/pyiron_atomistics/blob/main/pyiron_atomistics/atomistics/master/murnaghan.py#L958